File: 3600-find-the-k-th-character-in-string-game-i/3600-find-the-k-th-character-in-string-game-i.java
class Solution {
    public char kthCharacter(int k) {

        String word="a";
        char curr='a';
        while(word.length()<k){
            String nxt="";
            for(int i=0;i<word.length(); i++){
                char c=word.charAt(i);
                //int ascii=
                nxt+=(c=='z')?'a':(char)(c+1);
                //nxt+=nextChar;
            }
            word+=nxt;
        }
        return word.charAt(k-1);

        
        
    }
}

File: 3600-find-the-k-th-character-in-string-game-i/3600-find-the-k-th-character-in-string-game-i.cpp
class Solution {
public:
    char kthCharacter(int k) {

        string w="a";
        
        while(w.length()<k){
            string current="";
            for(char character:w){
                //char character=word.charAt(i);
               if( character=='z')current+'a';
               else current+=character+1;
                
            }
            w+=current;
        }
        return w[k-1];
        
    }
};